blackjack.py

```diff
@@ -106,14 +107,18 @@ def get_soft_hand_value(self):
         return (21.1 if self.check_natural_win()
                 else sum(card.points for card in self.hand))
 
-    def get_hard_hand_value(self):
+    def get_hand_value(self):
         """
-        Calculates hand value with Ace as 11
+        Calculates hand value with any Ace as 11 as long as it doesn't lead to bust
         :return: int
         """
-        return (21.1 if self.check_natural_win()
-                else self.get_soft_hand_value() +
-                10 * sum(card.value == 'Ace' for card in self.hand))
+        if self.check_natural_win():
```

Comment:
This method returns a boolean, so it probably should be named `is_natural_win`.

Reply:
done